runner/local/simple.go

```diff
@@ -62,11 +62,28 @@ func (r *simpleRunner) Run(cmd *runner.Command) (runner.ProcessStatus, error) {
 		return runner.ProcessStatus{}, fmt.Errorf(runner.RunnerBusyMsg)
 	}
 
+	// Redirect command output to the stdout/stderr Output vars. Outputs shall be closed in run().
```

Comment:
It's easy to do it here and falls under the category of job initialization shared across all jobs of all types. See status.go comment.